vlsi/example-tools.yml

```diff
@@ -0,0 +1,24 @@
+# SRAM Compiler compiler options
+vlsi.core.sram_generator_tool: "sram_compiler"
+# You should specify a location for the SRAM generator in the tech plugin
+vlsi.core.sram_generator_tool_path: []
+vlsi.core.sram_generator_tool_path_meta: "append"
+
+# Tool options. Replace with your tool plugin of choice.
+# Genus options
+vlsi.core.synthesis_tool: "genus"
+vlsi.core.synthesis_tool_path: ["hammer-cadence-plugins/synthesis"]
+vlsi.core.synthesis_tool_path_meta: "append"
+synthesis.genus.version: "1813"
+# Innovus options
+vlsi.core.par_tool: "innovus"
+vlsi.core.par_tool_path: ["hammer-cadence-plugins/par"]
+vlsi.core.par_tool_path_meta: "append"
+par.innovus.version: "191_ISR3"
```

Comment:
Is this used in the asap7 demo? If so it needs to be 181.

Reply:
The intention of this file is to be used as template for people using Hammer.
I did not want to start testing the ASAP7 demo again, and therefore I did not re-factor the ASAP7 tutorial yml (example-asap7.yml) and it is left as it previously was in it's monolithic form.